src/bin/wasm-tools/component.rs

```diff
@@ -107,6 +107,14 @@ pub struct NewOpts {
     /// Print the output in the WebAssembly text format instead of binary.
     #[clap(long, short = 't')]
     wat: bool,
+
+    /// Use memory.grow to realloc memory.
+    #[clap(long, short = 'r')]
+    realloc_via_memory_grow: bool,
+
+    /// Use memory.grow for stack allocation.
+    #[clap(long, short = 's')]
```

Comment:
I personally try to reserve `short` options for pretty common pieces of functionality, so I think it's ok to stick with just `long` for now and add `short` later if this ends up being more common.

Reply:
removed, thanks